### PR TITLE
Add option to create wpk for /opt and fix Solaris 10 log error

### LINKED ISSUE #385: Solaris 10 package contains error lines in log in a fresh install
Hi team,

We found two error lines in the `ossec.log` after doing a fresh install of the Solaris 10 package. Here you can see both lines:
```log
2020/01/28 13:37:03 ossec-agentd: ERROR: (4112): Invalid server address found: 'MANAGER_IP'
2020/01/28 13:37:03 ossec-agentd: CRITICAL: (1215): No client configured. Exiting.
```
Probably, the build script is starting the agent while the package is being built, those lines are logged in the log file and then, this log file is packaged with the rest of the files.

We must review and fix this.

### PR DESCRIPTION
**Pull request by DFolchA**
Hello team,

This PR closes #385, by fixing the cause of the error in the logs in Solaris10 clean installations, and add an option to the WPK generation to choose the installation path of Wazuh for the WPK.

Regards,
Daniel Folch

### CODE REVIEW

File: wpk/generate_wpk_package.sh [hunk 1]
```diff
@@ -15,40 +15,44 @@ LINUX_BUILDER_DOCKERFILE="${CURRENT_PATH}/unified/linux"
 WIN_BUILDER="windows_wpk_builder"
 WIN_BUILDER_DOCKERFILE="${CURRENT_PATH}/windows"
 CHECKSUM="no"
+INSTALLATION_PATH="/var"
 
 trap ctrl_c INT
 
 function build_wpk_windows() {
   local BRANCH="$1"
```
Comment (grg121) [inline wpk/generate_wpk_package.sh:23]: Please, try to the same format that below "${1}

File: wpk/generate_wpk_package.sh [hunk 2]
```diff
@@ -15,40 +15,44 @@ LINUX_BUILDER_DOCKERFILE="${CURRENT_PATH}/unified/linux"
 WIN_BUILDER="windows_wpk_builder"
 WIN_BUILDER_DOCKERFILE="${CURRENT_PATH}/windows"
 CHECKSUM="no"
+INSTALLATION_PATH="/var"
```
Comment (jmv74211) [inline wpk/generate_wpk_package.sh:18]: Maybe you could name this variable `DEFAULT_INSTALLATION_PATH`

File: wpk/generate_wpk_package.sh [hunk 3]
```diff
@@ -15,40 +15,44 @@ LINUX_BUILDER_DOCKERFILE="${CURRENT_PATH}/unified/linux"
...
 
 trap ctrl_c INT
 
 function build_wpk_windows() {
   local BRANCH="$1"
   local DESTINATION="$2"
```
Comment (jmv74211) [inline wpk/generate_wpk_package.sh:24]: Usa same format `"${x}"  or "$x"

File: wpk/generate_wpk_package.sh [hunk 4]
```diff
@@ -15,40 +15,44 @@ LINUX_BUILDER_DOCKERFILE="${CURRENT_PATH}/unified/linux"
...
-  local KEYDIR="$3"
-  local CONTAINER_NAME="$4"
-  local JOBS="$5"
-  local PACKAGE_NAME="$6"
-  local OUT_NAME="$7"
-  local CHECKSUM="$8"
-  local CHECKSUMDIR="$9"
+  local KEYDIR="${3}"
```
Comment (jmv74211) [inline wpk/generate_wpk_package.sh:25]: In this case, you could remove `{}`, like `KEYDIR="$3"` ?

File: wpk/generate_wpk_package.sh (hunk 1 above)
Comment (DFolchA) [reply wpk/generate_wpk_package.sh]: All unified to ${} https://github.com/wazuh/wazuh-packages/pull/408/commits/c331bd9e21fa8a17b41d971cd0348c35bdd54bd9

File: wpk/generate_wpk_package.sh (hunk 3 above)
Comment (DFolchA) [reply wpk/generate_wpk_package.sh]: All unified to ${} https://github.com/wazuh/wazuh-packages/pull/408/commits/c331bd9e21fa8a17b41d971cd0348c35bdd54bd9

File: wpk/generate_wpk_package.sh (hunk 4 above)
Comment (DFolchA) [reply wpk/generate_wpk_package.sh:25]: All unified to ${} https://github.com/wazuh/wazuh-packages/pull/408/commits/c331bd9e21fa8a17b41d971cd0348c35bdd54bd9

File: wpk/generate_wpk_package.sh (hunk 2 above)
Comment (DFolchA) [reply wpk/generate_wpk_package.sh]: Done in https://github.com/wazuh/wazuh-packages/pull/408/commits/c331bd9e21fa8a17b41d971cd0348c35bdd54bd9

File: wpk/generate_wpk_package.sh [hunk 5]
```diff
@@ -9,55 +9,59 @@
...
 # Foundation.
 
-CURRENT_PATH="$( cd $(dirname $0) ; pwd -P )"
+CURRENT_PATH="$( cd $(dirname ${0}) ; pwd -P )"
 LINUX_BUILDER="unified_linux_wpk_builder"
 LINUX_BUILDER_DOCKERFILE="${CURRENT_PATH}/unified/linux"
 WIN_BUILDER="windows_wpk_builder"
 WIN_BUILDER_DOCKERFILE="${CURRENT_PATH}/windows"
 CHECKSUM="no"
+lsION_PATH="/var"
```
Comment (BraulioV) [inline wpk/generate_wpk_package.sh:18]: This variable is unused.

File: wpk/generate_wpk_package.sh [hunk 6]
```diff
@@ -9,55 +9,59 @@
...
+  local BRANCH="${1}"
+  local DESTINATION="${2}"
+  local KEYDIR="${3}"
+  local CONTAINER_NAME="${4}"
+  local JOBS="${5}"
+  local PACKAGE_NAME="${6}"
+  local OUT_NAME="${7}"
+  local CHECKSUM="${8}"
+  local CHECKSUMDIR="${9}"
+  local DEFAULT_INSTALLATION_PATH="${10}"
```
Comment (BraulioV) [inline wpk/generate_wpk_package.sh:32]: Change the name of this variable to `INSTALLATION_PATH`. The default one is `/var/ossec` but if you can change it, it won't be the default anymore.

File: wpk/run.sh [hunk 7]
```diff
@@ -2,13 +2,14 @@
...
-BRANCH=$1
-JOBS=$2
-OUT_NAME=$3
-CHECKSUM=$4
-PKG_NAME=$5
+BRANCH=${1}
+JOBS=${2}
+OUT_NAME=${3}
+CHECKSUM=${4}
+DEFAULT_INSTALLATION_PATH=${5}
```
Comment (BraulioV) [inline wpk/run.sh:9]: Same here, change it to `INSTALLATION_PATH`.

File: wpk/generate_wpk_package.sh (hunk 5 above)
Comment (DFolchA) [reply wpk/generate_wpk_package.sh]: Fixed https://github.com/wazuh/wazuh-packages/pull/408/commits/266a8eff05ec92416c5eee4eb6f8adbfbf3a07ea

File: wpk/generate_wpk_package.sh (hunk 6 above)
Comment (DFolchA) [reply wpk/generate_wpk_package.sh]: Fixed https://github.com/wazuh/wazuh-packages/pull/408/commits/266a8eff05ec92416c5eee4eb6f8adbfbf3a07ea

File: wpk/run.sh (hunk 7 above)
Comment (DFolchA) [reply wpk/run.sh]: Fixed https://github.com/wazuh/wazuh-packages/pull/408/commits/266a8eff05ec92416c5eee4eb6f8adbfbf3a07ea

**Pull request merged**